bin/python/ripple/util/Sign.py

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env python
+
+from __future__ import print_function
+
+import base64, os, random, sys
+import ed25519
+import ecdsa
+from ripple.util import Base58
+from ripple.ledger import SField
+
+ED_BYTE = chr(0xed)
+
+USAGE = """\
+Usage:
+    create
+        Create a new master public/secret key pair.
+
+    check <key>
+        Check an existing key for validity.
+
+    sign <sequence> <validator-public> <master-secret>
+        Create a new signed manifest with the given sequence
+        number, validator public key, and master secret key.
+    """
+
+def to_bytes_bigend(n, length):
+    h = '%x' % n
+    return ('0' * (len(h) % 2) + h).zfill(length * 2).decode('hex')
+
+def prepend_length_byte(b):
+    assert len(b) < 192, 'Too long'
```

Comment:
Wait, I don't think that's right.  The TOTAL length needs to be no greater than 192 - including the length byte - yes?